### ISSUE TITLE
Open result in one file

### ISSUE THREAD

**Issue by fakhamatia** [fabiospampinato/vscode-search-open-all-results#1]
If search results in file more than one repeat when i use "Open all result" open file one time not per result.

![Screenshot from 2019-04-18 23-16-25](https://user-images.githubusercontent.com/20030910/56383944-62613200-6230-11e9-8c5a-3af8931d8e1c.png)

result is 16 in 8 file and open 8 tab not 16.

And sometimes error "Cannot read property 'model' of undefined" in VSCode corner.

**Comment by fabiospampinato** [comment]
If I'm understanding this correctly in the scenario showcased in the screenshot you'd like to have 16 tabs opened instead of 8?

> And sometimes error "Cannot read property 'model' of undefined" in VSCode corner.

Can you open a separate issue about this?

**Comment by fakhamatia** [comment]
> If I'm understanding this correctly in the scenario showcased in the screenshot you'd like to have 16 tabs opened instead of 8?
> 
> > And sometimes error "Cannot read property 'model' of undefined" in VSCode corner.
> 
> Can you open a separate issue about this?

**Comment by fakhamatia** [comment]
> If I'm understanding this correctly in the scenario showcased in the screenshot you'd like to have 16 tabs opened instead of 8?
> 
> > And sometimes error "Cannot read property 'model' of undefined" in VSCode corner.
> 
> Can you open a separate issue about this?

**Comment by F50Ci37Y** [comment]
> If I'm understanding this correctly in the scenario showcased in the screenshot you'd like to have 16 tabs opened instead of 8?

Yes , sorry for bad English. 

> > And sometimes error "Cannot read property 'model' of undefined" in VSCode corner.
> 
> Can you open a separate issue about this?

I dont know when happen this error , just error some times , open separate?

**Comment by fabiospampinato** [comment]
> Yes , sorry for bad English.

No problem, your English is ok. I'm afraid this is impossible to implement because of lacking APIs. We can't open the same file twice in separate tabs, unless those are side-by-side, but we can't have 10+ tabs opened in their own columns because the whole thing will become unusable.

> I dont know when happen this error , just error some times , open separate?

Yeah it would be better to open a separate issue about this.
